src/ansys/mapdl/core/cli/__init__.py

```diff
@@ -48,17 +48,19 @@ def main(ctx):
     main.add_command(stop)
     main.add_command(list_instances, name="list")
 
-    def old_pymapdl_convert_script_entry_point():
-        print(
-            """This CLI function has been deprecated. Please use instead:
-
-pymapdl convert input_file.inp -o output_file.out ...
-
-For more information please visit: https://mapdl.docs.pyansys.com/version/dev/user_guide/cli.html
-"""
-        )
 
 else:
 
     def main():
         print("PyMAPDL CLI requires 'click' python package to be installed.")
+
+
+def old_pymapdl_convert_script_entry_point():
+    print(
+        """This CLI function has been deprecated. Please use instead:
+
+pymapdl convert input_file.inp -o output_file.out ...
+
+For more information please visit: https://mapdl.docs.pyansys.com/version/dev/user_guide/cli.html
```

Comment:
```suggestion
Go to https://mapdl.docs.pyansys.com/version/dev/user_guide/cli.html for more information.
```
I don't like the suggested change because the reason to go there (for more information), should be first. However, I'm assuming we can't use a link like what follows? "For more information, see `PyMAPDL command line interface <https://mapdl.docs.pyansys.com/version/dev/user_guide/cli.html>`_."

Reply:
This is a message print to a terminal. Hence the link won't be rendered as in the docs. That's why I didn't use the link formatting. 

I'm addopting your suggestion though. Thank you!